jest.config.mjs

```diff
@@ -15,9 +15,21 @@
  *
  */
 const config = {
-  testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.(mjs?)$',
   moduleFileExtensions: ['js', 'mjs'],
   verbose: true,
+  testSequencer: './test/e2e/jestCustomSequencer.cjs',
```

Comment:
We need to see if we can avoid the need to run tests in sequence. Each test should "in theory" independent of each other as much as possible (each test should set its own environment and tear down afterwards).

Please create a ticket so that we can remove dependencies between tests.

Reply:
true, it was bugging me, too.

Created: https://github.com/hashgraph/solo/issues/62